### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kingobw**
Now we have a separate branch just for this called patch-1. Hope this one can merge.

### CODE REVIEW

**Comment by kingobw** [comment]
Talking to myself wearing both hats. This is FE the Owner of the master.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Hello!
-My name is FE King.
+No one is perfect. I, FE King, need to edit this file on my-branch.
+
```
Comment (kingobw) [inline README.md:3]: Add missing lines from Laura's presentation

File: README.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Hello!
-My name is FE King.
+No one is perfect. I, FE King, need to edit this file on my-branch.
```
Comment (kingobw) [inline README.md:2]:
```suggestion
No one is perfect. I, FE King, need to edit this file and add the repo on my-branch.


```

**Comment by kingobw** [comment]
Merging all good

**Comment by lsnovich** [comment]
You should add
FIX https://github.com/kingobw/trainingtest/issues/2
FIX https://github.com/kingobw/trainingtest/issues/1

**Pull request closed without merging**